### PR TITLE
Rename to ecdh-nistp256-kyber-512-sha256

### PR DESCRIPTION
**Pull request by anhu**
Used to be ecdh-sha2-nistp256-kyber-512-sha256 .

### CODE REVIEW

**Comment by csosto-pk** [comment]
@anhu , based on https://github.com/open-quantum-safe/openssh/issues/136 , I update the ```ecdh-nistp256-kyber-512-sha256``` to ```ecdh-nistp256-kyber-512r3-sha256-d00@openquantumsafe.org```. Can you update this PR as well?

File: src/internal.c [hunk 1]
```diff
@@ -1429,9 +1429,9 @@ static const NameIdPair NameIdMap[] = {
 #ifndef WOLFSSH_NO_DH_GEX_SHA256
     { ID_DH_GROUP14_SHA256, "diffie-hellman-group14-sha256" },
 #endif
-#ifndef WOLFSSH_NO_ECDH_SHA2_NISTP256_KYBER_LEVEL1_SHA256
+#ifndef WOLFSSH_NO_ECDH_NISTP256_KYBER_LEVEL1_SHA256
     /* We use kyber-512 here to achieve interop with OQS's fork. */
-    { ID_ECDH_SHA2_NISTP256_KYBER_LEVEL1_SHA256, "ecdh-sha2-nistp256-kyber-512-sha256" },
+    { ID_ECDH_NISTP256_KYBER_LEVEL1_SHA256, "ecdh-nistp256-kyber-512-sha256" },
```
Comment (csosto-pk) [inline src/internal.c:1434]: `ecdh-nistp256-kyber-512-sha256` should become `ecdh-nistp256-kyber-512r3-sha256-d00@openquantumsafe.org` as per https://github.com/open-quantum-safe/openssh/issues/136

File: src/internal.c [hunk 2]
```diff
@@ -7779,8 +7779,8 @@ static const char cannedMacAlgoNames[] =
 #endif
 
 static const char cannedKexAlgoNames[] =
-#if !defined(WOLFSSH_NO_ECDH_SHA2_NISTP256_KYBER_LEVEL1_SHA256)
-    "ecdh-sha2-nistp256-kyber-512-sha256,"
+#if !defined(WOLFSSH_NO_ECDH_NISTP256_KYBER_LEVEL1_SHA256)
+    "ecdh-nistp256-kyber-512-sha256,"
```
Comment (csosto-pk) [inline src/internal.c:7783]: `ecdh-nistp256-kyber-512-sha256` should become `ecdh-nistp256-kyber-512r3-sha256-d00@openquantumsafe.org` as per https://github.com/open-quantum-safe/openssh/issues/136

**Review by csosto-pk — COMMENTED** [review]
Suggested changes as per https://github.com/open-quantum-safe/openssh/issues/136

**Comment by anhu** [comment]
Hi @csosto-pk (Panos),

For now, I need to keep it this way so I can continue working with what is currently on master of that repo. 
That said, for sure, I will change it over when those PRs get merged.  I will likely wait for those PR to merge before I merge this one. 

Warm regards, @anhu (Anthony)

**Comment by anhu** [comment]
This is closed because https://github.com/wolfSSL/wolfssh/pull/510 replaces it.

**Pull request closed without merging**